entrypoint.sh

```diff
@@ -13,12 +13,12 @@ if [ $(echo ${INPUT_TITLE} | wc -c) -eq 1 ] ; then
 fi
 
 printf "\u001b[30;40m♫♪.ılılıll|̲̅̅●̲̅̅|̲̅̅=̲̅̅|̲̅̅●̲̅̅|%s|̲̅̅●̲̅̅|̲̅̅=̲̅̅|̲̅̅●̲̅̅|llılılı.♫♪\u001b[0m\n" "milestone"
-printf "\u001b[30;40m--------{---(@%s@}}>-----\u001b[0m\n" ${INPUT_TITLE}
+printf "\u001b[30;40m--------{---(@%s@}}>-----\u001b[0m\n" "${INPUT_TITLE}"
 
 if [ $(echo ${INPUT_DESCRIPTION} | wc -c) -gt 1 ] ; then
```

Comment:
Unsure if this also needs quoting

```suggestion
if [ $(echo "${INPUT_DESCRIPTION}" | wc -c) -gt 1 ] ; then
```

Reply:
Same with the input title empty check on line 10

Reply:
@sc0ttkclark Go for it, think it's wiser to do anyway. Once this is merged and released I'll figure out a way to test the API calls to GitHub to unit test this thing to be sure it fully works as intended.

Reply:
I'll get those done today for you. Thanks for being so receptive to the PR 😄 

I have really enjoyed being able to have an action for this to automate more processes.

Reply:
Adjustments done

Reply:
> I'll get those done today for you. Thanks for being so receptive to the PR 😄

Sweet thanks, will start working on the testing updates later tonight or tomorrow before relating these changes.

> I have really enjoyed being able to have an action for this to automate more processes.

Let me know if you want to bounce some ideas off me for that. Got my entire release process down to the click of closing a milestone. Having a milestone added to every PR is key to that, this is the or one of the first Actions I've created.

Reply:
Oh snap I didn't even consider that. Setting up a new release/milestone on close would be pretty cool too. I'll explore that.